src/main/java/emt/tile/TileEntityEtherealMacerator.java

```diff
@@ -226,18 +225,17 @@ else if (slots[i].stackSize < 63)
         }
     }
 
-    public boolean isOP() {
-        final int cobblestoneId = OreDictionary.getOreID("blockCobble");
+    public boolean isAllowed() {
 
-        if (OreDictionary.getOreID(this.slots[0]) == cobblestoneId || this.slots[0].getItem() == Item.getItemFromBlock(Blocks.sand) || this.slots[0].getItem() == Items.potato || this.slots[0].getItem() == Items.porkchop || this.slots[0].getItem() == Items.beef || this.slots[0].getItem() == Items.chicken || this.slots[0].getItem() == Items.fish
-                || this.slots[0].getItem() == IC2Items.getItem("coalDust").getItem()
-                || (OreDictionary.getOreName(OreDictionary.getOreID(this.slots[0])).toLowerCase().contains("dust") || this.slots[0].getItem().getUnlocalizedName(this.slots[0]).toLowerCase().contains("armour") || this.slots[0].getItem().getUnlocalizedName(this.slots[0]).toLowerCase().contains("armor") || this.slots[0].getItem().getUnlocalizedName(this.slots[0]).toLowerCase().contains("helm")
-                || this.slots[0].getItem().getUnlocalizedName(this.slots[0]).toLowerCase().contains("legging") || this.slots[0].getItem().getUnlocalizedName(this.slots[0]).toLowerCase().contains("boot") || this.slots[0].getItem().getUnlocalizedName(this.slots[0]).toLowerCase().contains("chestp") || this.slots[0].getItem().getUnlocalizedName(this.slots[0]).toLowerCase().contains("pick")
-                || this.slots[0].getItem().getUnlocalizedName(this.slots[0]).toLowerCase().contains("shovel") || this.slots[0].getItem().getUnlocalizedName(this.slots[0]).toLowerCase().contains("axe") || this.slots[0].getItem().getUnlocalizedName(this.slots[0]).toLowerCase().contains("pick"))) {
-            return true;
-        } else {
-            return false;
+        int[] OreDictIDs = OreDictionary.getOreIDs(this.slots[0]);
+        for (int ID : OreDictIDs) {
```

Comment:
Please use the naming style consistent with surrounding code. This repo is already filled with code smell, so let's don't add another.

Reply:
While I put the comment on this line, it doesn't mean this line is the only line with this problem. The existing code in this project uses lower camel case, so please stick to that.

Reply:
ok